src/components/collapsible/Collapsible.js

```diff
@@ -28,8 +28,12 @@ type Props = {
     isBordered?: boolean,
     /** Initial state of the collapsible card */
     isOpen: boolean,
+    /** callback called when collapsible is unfocused */
+    onBlur?: Function,
```

Comment:
If these are needed, for new types, I would add full typings like () => {...} instead of just Function

Reply:
have removed onFocus and onBlur